pkg/descheduler/policyconfig.go

```diff
@@ -137,11 +137,6 @@ func validateDeschedulerConfiguration(in api.DeschedulerPolicy, registry pluginr
 	for _, profile := range in.Profiles {
 		for _, pluginConfig := range profile.PluginConfigs {
 			if _, ok := registry[pluginConfig.Name]; ok {
-				if _, ok := registry[pluginConfig.Name]; !ok {
```

Comment:
Right :)

This could be actually moved one level up:
```go
for _, pluginConfig := range profile.PluginConfigs {
	if _, ok := registry[pluginConfig.Name]; !ok {
		errorsInProfiles = fmt.Errorf("%w: %s", errorsInProfiles, fmt.Sprintf("in profile %s: plugin %s in pluginConfig not registered", profile.Name, pluginConfig.Name))
		continue
	}
	pluginUtilities := registry[pluginConfig.Name]
	...
```
to remove another level of nesting.

Reply:
this is actually unrelated to the fix, but just seemed really odd. I can change it but it will show up as a lot more lines in +/-